packages/logger/src/logBuffer.ts

```diff
@@ -17,6 +17,7 @@ export class SizedItem<V> {
 
 export class SizedSet<V> extends Set<SizedItem<V>> {
   public currentBytesSize = 0;
+  public evictedLogCount = 0;
```

Comment:
I dislike that this is converting a more generic class to something more specific, but extending the SizedSet in a new class (`Buffer`?) seemed like overkill.

Reply:
Why are we keeping the count and incrementing it instead of just having a `boolean`? What's the advantage?